internal/pkg/manifest/job.go

```diff
@@ -135,6 +135,11 @@ func (j ScheduledJob) ApplyEnv(envName string) (WorkloadManifest, error) {
 	return &j, nil
 }
 
+// TaskPlatform returns the os/arch for the service. This is an empty string if the default (linux/amd64) is detected.
```

Comment:
`t.Platform` is read a user input read from manifest right 🤔 ? I may be missing something from previous PRs - we don't do anything detecting after reading in the manifest right?

Reply:
Yes. Changed the comment.